cmd/notify-mailer/main.go

```diff
@@ -456,10 +457,26 @@ func main() {
 	defer log.AuditPanic()
 
 	// Setup database client.
-	dbURL, err := cfg.NotifyMailer.DB.URL()
+	dsn, err := cfg.NotifyMailer.DB.URL()
```

Comment:
Any particular reason you're changing this here but not elsewhere? We use `dbURL` pretty ubiquitously, I think.

Reply:
Good point.